### ISSUE TITLE
Tabulate Error with --help-all

### ISSUE THREAD

**Issue by TheTechromancer** [blacklanternsecurity/bbot#364]
~~~python
[ERRR] bbot: Encountered unknown error: Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/bbot/cli.py", line 223, in main
    for row in module_loader.modules_options_table(modules=help_modules).splitlines():
  File "/usr/local/lib/python3.10/site-packages/bbot/core/helpers/modules.py", line 291, in modules_options_table
    return make_table(table, header)
  File "/usr/local/lib/python3.10/site-packages/bbot/core/helpers/misc.py", line 755, in make_table
    return tabulate(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/tabulate/__init__.py", line 2061, in tabulate
    list_of_lists = _wrap_text_to_colwidths(
  File "/usr/local/lib/python3.10/site-packages/tabulate/__init__.py", line 1518, in _wrap_text_to_colwidths
    wrapped = wrapper.wrap(casted_cell)
  File "/usr/local/lib/python3.10/textwrap.py", line 359, in wrap
    chunks = self._split_chunks(text)
  File "/usr/local/lib/python3.10/textwrap.py", line 345, in _split_chunks
    text = self._munge_whitespace(text)
  File "/usr/local/lib/python3.10/textwrap.py", line 154, in _munge_whitespace
    text = text.expandtabs(self.tabsize)
AttributeError: 'bool' object has no attribute 'expandtabs'
~~~

**Comment by TheTechromancer** [comment]
Upstream issue in tabulate: https://github.com/astanin/python-tabulate/issues/189

**Comment by TheTechromancer** [comment]
Temporarily fixed by downgrading tabulate to `0.8.10` in https://github.com/blacklanternsecurity/bbot/pull/369

**Comment by TheTechromancer** [comment]
This bug was fixed by downgrading, no need to keep open.
